### PR TITLE
Parsing all cases for adjective declension (#717)

### PR DESCRIPTION
**Pull request by psfinaki**
I want to create a new exercise for general adjective declension and for that we need to take all the cases from wiki.

I implemented it in similar fashion to what we have with nouns: created a separate module (`AdjectiveWikiParsing`) doing just parsing stuff and integrated it into `AdjectiveArticle`.

### CODE REVIEW

File: src/WikiParsing/AdjectiveWikiParsing.fs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        PluralNominative =     data.Tables.``Skloňování[editovat]``.Rows.[0].``plurál - mužský životný``  
+        PluralGenitive =       data.Tables.``Skloňování[editovat]``.Rows.[1].``plurál - mužský životný``  
+        PluralDative =         data.Tables.``Skloňování[editovat]``.Rows.[2].``plurál - mužský životný``  
+        PluralAccusative =     data.Tables.``Skloňování[editovat]``.Rows.[3].``plurál - mužský životný``  
+        PluralVocative =       data.Tables.``Skloňování[editovat]``.Rows.[4].``plurál - mužský životný``  
+        PluralLocative =       data.Tables.``Skloňování[editovat]``.Rows.[5].``plurál - mužský životný``  
+        PluralInstrumental =   data.Tables.``Skloňování[editovat]``.Rows.[6].``plurál - mužský životný``  
+    }
+
+let get2Declension (AdjectiveArticleWithPlural (AdjectiveArticle { Text = text })) =
```
Comment (reflash) [inline src/WikiParsing/AdjectiveWikiParsing.fs:56]: What's get1, get2... mean? Is it different page structures?

File: src/WikiParsing/AdjectiveWikiParsing.fs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    let data = WikiAdjectiveStarý.Parse text
+    {
+        SingularNominative =   data.Tables.``Skloňování[editovat]2``.Rows.[0].``singulár - mužský životný``
```
Comment (reflash) [inline src/WikiParsing/AdjectiveWikiParsing.fs:59]:
```fsharp
let rows = data.Tables.``Skloňování[editovat]2``.Rows
...
SingularNominative =   rows.[0].``singulár - mužský životný``
```

Comment (psfinaki) [reply src/WikiParsing/AdjectiveWikiParsing.fs]: Good point - `rows` is probably not that expressive though so I used captured `table` instead.

File: src/WikiParsing/AdjectiveWikiParsing.fs (hunk 1 above)
Comment (psfinaki) [reply src/WikiParsing/AdjectiveWikiParsing.fs]: It's bad naming then :)

Well, the thing is that some adjectives can be also nouns (e.g. `vrchní`) and to get the actual adjective declension we need to parse the last declension table on the page. So it is a simple idea (a bit subtle also but well) - if the word can be an adjective then adjective declension will be in the last declension table on the page.

Compare [nový](https://cs.wiktionary.org/wiki/nov%C3%BD) vs [vrchní](https://cs.wiktionary.org/wiki/vrchn%C3%AD). Check if the new naming is any clearer...

**Review by reflash — APPROVED** [review]

**Pull request merged**